src/adaptors/UBMetadataDcSubsetAdaptor.cpp

```diff
@@ -31,6 +31,8 @@
 
 #include <QtGui>
 #include <QtXml>
+#include <QGuiApplication>
+#include <QScreen>
```

Comment:
These two added `#include`s are actually not needed, I think.

Reply:
Oh right, the code that used them got lost during rebasing, as the original code was already replaced.